src/packer.js

```diff
@@ -91,7 +91,7 @@ const Packer = new Lang.Class ({
     },
 
     _add_set: function (metadata) {
-        let set_hash = this._hashify(metadata['title']);
+        let set_hash = this._hashify(metadata['source']);
```

Comment:
Kind of, it was a quick hack to avoid having to rely on the set tittle (since it can be repeated e.g. in a hierarchy). But,I will fix that later.